addons/hr_expense/views/hr_expense_views.xml

```diff
@@ -624,8 +624,9 @@
                     <field name="total_amount" optional="show" sum="Total Amount" decoration-bf="True"/>
                     <field name="currency_id" optional="hide"/>
                     <field name="journal_id" optional="hide"/>
-                    <field name="state" optional="show" decoration-info="state == 'draft'" decoration-success="state in ['submit', 'approve', 'post', 'done']" decoration-danger="state == 'cancel'" widget="badge"/>
```

Comment:
not sure we should remove the `state` field

Reply:
indeed. Can you tranche @tsb-odoo ?

Reply:
@qdp-odoo @smetl Let's keep it.